extensions/odoo_theme/static/style.scss

```diff
@@ -997,6 +997,99 @@ header.o_main_header {
         }
     }
 }
+//------------------------------------------------------------------------------
+// Content Tabs
+//------------------------------------------------------------------------------
+
+@mixin o-content-tab-selected {
+    font-weight: 700;
+    background: $doc-paper;
+    border-bottom: 1px solid $doc-paper;
+    border-left: 1px solid $gray-light;
+    border-right: 1px solid $gray-light;
+}
+
+.sphinx-tabs {
+    padding: 0;
+    margin-bottom: 1rem;
+
+    div[role^="tablist"] {
+        border-color: $gray-light;
+        display: grid;
+        width: 100%;
+        grid-template-columns: repeat(6, minmax(6.65rem, 1fr));
+    }
+
+    *:focus-visible {
+        outline: none;
+    }
+
+    button {
+        display: inline;
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    .sphinx-tabs-tab {
+        background: $gray-lightest;
+        color: $gray-900;
+        border: 0;
+        border-top: solid 1px $gray-light;
+        padding: 1rem 1.5rem;
```

Comment:
I would reduce the padding. 
`padding: .5rem 1rem;`

Reply:
Done!